ui/packages/atlasmap/src/Atlasmap/dialogs/useConstantDialog.tsx

```diff
@@ -33,6 +33,7 @@ export function useConstantDialog(
         value,
         label,
       }))}
+      valueType={constantTypes[0][0]}
```

Comment:
We have `initialConstant` to set the default, which is set where the dialog is actually set up instead of here. I'd rather keep this logic there.

Reply:
Thanks for the feedback @riccardo-forina, I updated the PR to utilize the `initialConstant`, could you please have a look?